subiquity/common/filesystem/labels.py

```diff
@@ -258,19 +258,36 @@ def for_client(device, *, min_size=0):
 @for_client.register(Disk)
 @for_client.register(Raid)
 def _for_client_disk(disk, *, min_size=0):
+    path = disk.path if getattr(disk, 'path', None) else None
```

Comment:
How does this differ from "path = getattr(disk, 'path', None)"? I guess it's different if disk.path is a non-None falsey value but surely that's not what this is about?